docs/content/direct/release-notes.md

```diff
@@ -4,6 +4,9 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.21.0 and its release candidates
 
+### Patch release  0.21.1 
```

Comment:
"Patch release  0.21.1 " does not logically fit under "0.21.0 and its release candidates". BTW, even for a patch release we will have release candidates. I think the best approach is to simply have "## 0.21.1 and its release candidates".

Reply:
Done